modules/machine-config-node-disruption-config.adoc

```diff
@@ -0,0 +1,138 @@
+
+You can control how your nodes respond to changes in the configuration files in the `/var` or `/etc` directory, systemd units, or SSH keys.
+
+include::snippets/machine-config-node-disruption-actions.adoc[]
+
+.Prerequisite
+
+
+* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see _Enabling features using feature gates_.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster prevents minor version updates. The `TechPreviewNoUpgrade` feature set cannot be disabled. Do not enable this feature set on production clusters.
+====
+
+.Procedure
+
+. Edit the `MachineConfiguration` object to define the node disruption policy:
++
+[source,terminal]
+----
+$ oc edit MachineConfiguration cluster -n openshift-machine-config-operator
+----
+
+. Add a node disruption policy similar to the following:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: MachineConfiguration
+metadata:
+  name: cluster
+  namespace: openshift-machine-config-operator
+  labels:
+    machineconfiguration.openshift.io/role: worker # <1>
+# ...
+spec:
+  nodeDisruptionPolicy: <2>
+    files: # <3>
+    - actions: # <4>
+      - reload: # <5>
+          serviceName: chronyd.service # <6>
+        type: Reload
+      path: /etc/chrony.conf # <7>
+    sshkey: # <8>
+      actions:
+      - type: Drain
+      - reload:
+          serviceName: crio.service
+        type: Reload
+      - type: DaemonReload
+      - restart:
+          serviceName: crio.service
+        type: Restart
+    units: # <9>
+    - actions:
+      - type: Drain
+      - reload:
+          serviceName: crio.service
+        type: Reload
+      - type: DaemonReload
+      - restart:
+          serviceName: crio.service
+        type: Restart
+      name: test.service
+----
+<1> Specifies the role label for the custom machine config pool where you want to use the policy.
+<2> Specifies the node disruption policy for the nodes associated with the role label you configured. 
+<3> Specifies a list of machine config file definitions and actions to take to changes on those paths This list supports a maximum of 50 entries.
+<4> Specifies the series of action to be executed upon changes to the specified files. Actions are applied in the order that they are set in this list. This list supports a maximum of 10 entries.
+<5> Specifies that the specified service is to be reloaded upon changes to the specified files. 
+<6> Specifies the full name of the service to be acted upon. 
+<7> Specifies the location of a file that is managed by a machine config. The actions in the policy apply when changes are made to the file in `path`.
+<8> Specifies an action all changes to the SSH key changes in the cluster.
+<9> Specifies a list of systemd unit names and actions to take upon changes to those units. This list supports a maximum of 50 entries.
+
+.Verification
+
+. View the `MachineConfig` object file that you created:
```

Comment:
```suggestion
. View the `MachineConfiguration` object file that you updated:
```
A default MachineConfiguration object will exist before any changes are applied, showing the standard MCO policies. It may be good to point those out here as well. More info here: https://github.com/openshift/machine-config-operator/blob/master/docs/NodeDisruptionPolicy.md#how-it-works-right-now
A user will always be updating it with their custom policies(overriding these standard policies, if there is a conflict)